data/remnant/remnant.txt

```diff
@@ -276,6 +276,498 @@ phrase "remnant"
 		"Soaring"
 		"Enduring"
 
+phrase "remnant uncontacted"
+	word
+		"(They sent you a hail consisting of a sign language you don't understand.)"
+
+phrase "remnant uncontacted"
+	word
+		"I am"
+		"We are"
+	word
+		" "
+	word
+		"watching"
+		"observing"
+		"analyzing"
+		"overseeing"
+	word
+		" "
+	word
+		"you"
+		"your actions"
+		"your behavior"
+	word
+		"."
+
+phrase "remnant uncontacted hostile"
+	word
+		"Eat space dust"
+		"Kill yourself"
+		"Sing your last prayers"
+		"Leave this space"
+	word
+		","
+	word
+		" "
+	word
+		"hostile ship"
+		"pirate"
+		"genetically modified biological weapon"
```

Comment:
```suggestion
		"Alpha"
```
Don't see what the point of using `genetically modified biological weapon` over `Alpha` is.